server/queries/findAdminByCredentials.test.js

```diff
@@ -1,43 +1,41 @@
 /* eslint-disable no-undef */
-// const { ObjectID } = require('mongodb');
-// const { initDb, closeDb } = require('../databaseConnection');
+const { initDb, closeDb } = require('../databaseConnection');
 
-// const findAdminByCredentials = require('./findAdminByCredentials');
+const findAdminByCredentials = require('./findAdminByCredentials');
 
-it('Test to pass file', (done) => {
-  done();
+beforeEach(() => {
+  return initDb();
```

Comment:
In server/queries/addQuestions.test.js, you call initDb() inside the test but here, you are using beforeEach().